governance/votes/Executive vote - June 7, 2023.md

```diff
@@ -0,0 +1,215 @@
+- Increase the WBTC-A Stability Fee by 0.9% from 4.90% to **5.80%**.
+- Increase the WBTC-B Stability Fee by 1.4% from 4.90% to **6.30%**.
+- Increase the WBTC-C Stability Fee by 0.65% from 4.90% to **5.55%**.
+
+### USDP PSM [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) Reduction
+
+As per [this poll](https://vote.makerdao.com/polling/QmQYSLHH), the following change to the USDP PSM Debt Ceiling will be made, if this executive proposal passes:
+- Reduce the USDP PSM (technical name PSM-PAX-A) Debt Ceiling by 500 million DAI from 500 million DAI to **0**.
+
+*Technical implementation note:* to achieve this, the USDP PSM will be removed from the Autoline (DC-IAM) and have its Debt Ceiling set to 0.
+
+### MKR Vesting Transfers
+
+The following MKR vesting transfers will be made, if this executive proposal passes:
+
+#### Sidestream
+
+As per their [approved budget](https://mips.makerdao.com/mips/details/MIP40c3SP44), **348.28 MKR** will be transferred to the Sidestream Auditor Wallet at [0xb1f950a51516a697E103aaa69E152d839182f6Fe](https://etherscan.io/address/0xb1f950a51516a697E103aaa69E152d839182f6Fe).
+
+*Note:* the distributed amount has differed from the initial estimate due to Sidestream [repricing](https://forum.makerdao.com/t/mip40c3-sp44-adding-sidestream-auction-services-core-unit-mkr-budget-sas-001/10802/7) their vesting plan. It remains within the overall approved cap.
+
+#### DUX
+
+As per their [approved budget](https://mips.makerdao.com/mips/details/MIP40c3SP27), **225.12 MKR** will be transferred to the DUX Auditor Wallet at [0x5A994D8428CCEbCC153863CCdA9D2Be6352f89ad](https://etherscan.io/address/0x5A994D8428CCEbCC153863CCdA9D2Be6352f89ad).
+
+*Note:* the distributed amount has differed from the initial estimate due to additional hires. It remains within the overall approved cap.
+
+### Spark Protocol [Dai Direct Deposit Module](https://manual.makerdao.com/module-index/module-dai-direct-deposit) (D3M) Parameter Changes
+
+As per [this poll](https://vote.makerdao.com/polling/QmWatYqy), the following Spark Protocol D3M parameter changes will be made, if this executive proposal passes:
+- Increase the [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam) by 15 million DAI from 5 million DAI to **20 million DAI**.
+- Increase the [Target Availible Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) by 15 million DAI from 5 million DAI to **20 million DAI**.
+
+### Spark Protocol Proxy (`EXEC_PROXY_SPARK`) Authorization
+
+As per [this poll](https://vote.makerdao.com/polling/Qmc9fd3j), a Spark Protocol Proxy will be authorized by this spell to allow parallel production of executive proposals covering Spark Protocol. This will allow the Spark Protocol team to prepare code separately, but any changes will not be active until triggered by a Maker Core executive proposal.
```

Comment:
Can link to here again somewhere? <https://forum.makerdao.com/t/executive-proxies-and-spark-protocol/20976>